src/iopae.cpp

```diff
@@ -233,7 +233,7 @@ void PaeInput::parsePlainAndEasy(std::istream &infile)
         }
 
         // rhythmic values
-        else if ((isdigit(incipit[i]) != 0) && (incipit[i-1] != ';')) {
+        else if ((isdigit(incipit[i]) != 0) && ((i = 0) || incipit[i-1] != ';')) {
```

Comment:
Fix it with
`else if ((isdigit(incipit[i]) != 0) && ((i <= 0) || incipit[i-1] != ';')) {`

Reply:
@lpugin I tried to force an index error without any luck, and then I found this answer: https://stackoverflow.com/questions/1239938/accessing-an-array-out-of-bounds-gives-no-error-why

Reply:
Yes, with your code it could happen that you would read outside the range of the array. Depending how it is allocated, this can go through. But then anything can be there...

Everywhere else in Verovio we use `std::vector` and `at()` method, which will raise an error when accessing outside the range. This makes it easier to find a problem.